inginious/frontend/tests/test_course.py

```diff
@@ -0,0 +1,122 @@
+# pylint: disable=redefined-outer-name
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+from collections import OrderedDict
+
+import pytest
+import os
+import tempfile
+import shutil
+
+from inginious.common.filesystems.local import LocalFSProvider
+from inginious.frontend.course_factory import create_factories
+from inginious.common.tasks_problems import *
+from inginious.frontend.task_dispensers.toc import TableOfContents
+from inginious.frontend.environment_types import register_base_env_types
+from inginious.frontend.task_dispensers.combinatory_test import CombinatoryTest
+
+task_dispensers = {TableOfContents.get_id(): TableOfContents, CombinatoryTest.get_id(): CombinatoryTest}
+problem_types = {"code": CodeProblem, "code_single_line": CodeSingleLineProblem, "file": FileProblem,
+                 "multiple_choice": MultipleChoiceProblem, "match": MatchProblem}
+
+
+@pytest.fixture(params=[LocalFSProvider])
+def ressource(request):
+    register_base_env_types()
+    dir_path = tempfile.mkdtemp()
+    fs = LocalFSProvider(os.path.join(os.path.dirname(__file__), 'tasks'))
```

Comment:
I don't understand why you parameterize the fixture with `LocalFSProvider`, then initialize a `LocalFSProvider` whatever the `request.param` will be, then yield the `request.param` and never use it in any of the tests in the file...